src/components/ContentNode.vue

```diff
@@ -227,14 +229,13 @@ function renderNode(createElement, references) {
       };
       return createElement(EndpointExample, { props }, renderChildren(node.summary || []));
     }
-    case BlockType.heading:
-      return createElement(`h${node.level}`, {
-        attrs: {
-          id: node.anchor,
-        },
-      }, (
-        node.text
-      ));
+    case BlockType.heading: {
+      const props = {
+        anchor: node.anchor,
+        tag: `h${node.level}`,
```

Comment:
I think maybe we should pass the `level` value as a prop and then map those to the right header tag in the new component—otherwise we allow for any arbitrary string to be used as the HTML tag for this component when we probably only want to allow h1-h6.

Reply:
done!